tests/test_assets/gene_info/input/ensg_to_uniprot_mapping_good.tsv

```diff
@@ -0,0 +1,18462 @@
+uniprotkb_accessions	ensembl_gene_id
```

Comment:
Is there any way you can shorten this file to just a few accession/ensembl pairs? Otherwise it may be hard to add to / edit this test case later.

Reply:
Can do!

Reply:
Thanks. Are there any repeat Ensembl IDs in this file? If not, I think it's worth adding at least one.